### PR TITLE
Optionally install ceph-test

### PR DESCRIPTION
**Pull request by zmc**
We want this to enable testing with existing teuthology jobs.

https://github.com/ceph/teuthology

Signed-off-by: Zack Cerza <zack@redhat.com>

### CODE REVIEW

File: roles/ceph-common/tasks/installs/install_on_redhat.yml [hunk 1]
```diff
@@ -71,6 +71,18 @@
     osd_group_name in group_names and
     ansible_pkg_mgr == "dnf"
 
+- name: install ceph-test
+  yum:
+    name: ceph-test
+    state: "{{ (upgrade_ceph_packages|bool) | ternary('latest','present') }}"
+  when: ceph_test and ansible_pkg_mgr == "yum"
```
Comment (leseb) [inline roles/ceph-common/tasks/installs/install_on_redhat.yml]: can you apply the following syntax style?

```
when:
  ceph_test and
  ansible_pkg_mgr == "yum"
```

It is easier to read :)

File: roles/ceph-common/tasks/installs/install_on_redhat.yml [hunk 2]
```diff
@@ -71,6 +71,18 @@
...
+
+- name: install ceph-test
+  dnf:
+    name: ceph-test
+    state: "{{ (upgrade_ceph_packages|bool) | ternary('latest','present') }}"
+  when: ceph_test and ansible_pkg_mgr == "dnf"
```
Comment (leseb) [inline roles/ceph-common/tasks/installs/install_on_redhat.yml]: dito

File: roles/ceph-common/tasks/installs/install_on_redhat.yml (hunk 1 above)
Comment (zmc) [inline roles/ceph-common/tasks/installs/install_on_redhat.yml]: Absolutely!

File: roles/ceph-common/tasks/installs/install_on_redhat.yml (hunk 2 above)
Comment (zmc) [inline roles/ceph-common/tasks/installs/install_on_redhat.yml]: ack

**Comment by leseb** [comment]
test this please

**Pull request merged**